### PR TITLE
Update paper.bib and paper.md

### PR DESCRIPTION
**Pull request by guanfada**
@SimonBiggs my new co-authored paper added.

### CODE REVIEW

File: paper.bib [hunk 1]
```diff
@@ -37,6 +37,7 @@ @phdthesis{rembish2021automating
   title={Automating Medical Physics Quality Assurance Tasks in Radiation Oncology},
   author={Rembish, Jacob},
   year={2021},
+  url={https://www.proquest.com/docview/2564568968}
```
Comment (Matthew-Jennings) [inline paper.bib:40]:
```suggestion
  url={https://www.proquest.com/docview/2564568968},
```

File: paper.bib [hunk 2]
```diff
@@ -91,6 +92,7 @@ @article{cronholm2020mri
   title={MRI only radiotherapy planning using the transfer function estimation algorithm},
   author={Cronholm, Rickard O and Karlsson, Anders and Siversson, Carl},
   year={2020}
```
Comment (Matthew-Jennings) [inline paper.bib:94]:
```suggestion
  year={2020},
```

**Review by Matthew-Jennings — COMMENTED** [review]
The paper PDF was unable to build (see [the correspondingly failing CI output](https://github.com/pymedphys/pymedphys/pull/1700/checks#step:4:12)), likely as a result of the missing commas.

You can see and interrogate failing tests at the bottom of the "Conversation" tab of the PR here:

image

Some of the tests in that list may not (yet) mean much to you, but it's telling if the Paper Draft one fails (as opposed to cancels). You can click "Details" next to the Paper Draft test to explore reasons for the test failing. This one is probably fairly easily diagnosable once one knows where to look (and assuming no other issues)!

**Comment by Matthew-Jennings** [comment]
Hi @guanfada,

The Paper Draft test is still failing with a very similar error (not the same!) as before. Do you want to have a go at fixing it using my instructions above?

**Comment by SimonBiggs** [comment]
Also, @guanfada, if you have docker, you can utilise the following approach to test the build locally:

https://joss.readthedocs.io/en/latest/submitting.html#docker

**Comment by guanfada** [comment]
> Hi @guanfada,
> 
> The Paper Draft test is still failing with a very similar error (not the same!) as before. Do you want to have a go at fixing it using my instructions above?

@Matthew-Jennings , yes, it is a similar comma missing error. I added it.

**Comment by guanfada** [comment]
> Also, @guanfada, if you have docker, you can utilise the following approach to test the build locally:
> 
> https://joss.readthedocs.io/en/latest/submitting.html#docker

@SimonBiggs I am not familiar with docker. :-(

**Comment by Matthew-Jennings** [comment]
@guanfada, do you have `pre-commit` installed in your local `pymedphys` directory?  E.g., see the [Windows setup guide ](https://docs.pymedphys.com/en/latest/contrib/setups/setup-win.html#install-the-development-version-of-pymedphys-and-pre-commit) for contributors.

If you haven't installed `pre-commit`, its installation may fix the failing pre-commit test on Github. Once installed, you make some dummy change to a file (e.g. add a new line somewhere) and commit the change. On commit, pre-commit should automatically be run and fix any issues in this branch that violate the pre-commit checks (I suspect the trailing whitespace in `paper.md`)

**Comment by guanfada** [comment]
> @guanfada, do you have `pre-commit` installed in your local `pymedphys` directory? E.g., see the [Windows setup guide ](https://docs.pymedphys.com/en/latest/contrib/setups/setup-win.html#install-the-development-version-of-pymedphys-and-pre-commit) for contributors.
> 
> If you haven't installed `pre-commit`, its installation may fix the failing pre-commit test on Github. Once installed, you can make some dummy change to a file (e.g. add a new line somewhere) and commit the change. When you commit, `pre-commit` should automatically run and fix any issues in the local branch that violate the pre-commit checks (I suspect the trailing whitespace in `paper.md`). Commit any new changes introduced by the `pre-commit` auto-run and push them.

I have below
image

**Comment by Matthew-Jennings** [comment]
> I have below

Hmm. Let me take a look.

**Comment by Matthew-Jennings** [comment]
> BTW, I found the PyMedPhys in my local computer is not synchronized with GitHub.
I have used Pip install --upgrade pymedphys, and it showed my current version is 0.38.0-dev4
but the Readme.rst in my local computer is still the version of December 7, 2021.
Any idea?

I think both this and the pre-commit issue might be related to the fork-and-pull workflow you're utilising in this PR/branch (it's trying to merge from `guanfada:main`. Whereas, I notice that you've done a direct push to a branch of `pymedphys/pymedphys:guanfada-patch-1` in #1701.

**Comment by SimonBiggs** [comment]
> BTW, I found the PyMedPhys in my local computer is not synchronized with GitHub.
> I have used Pip install --upgrade pymedphys, and it showed my current version is 0.38.0-dev4
> but the Readme.rst in my local computer is still the version of December 7, 2021.
> Any idea?

This is actually by design. When following the setup guide:

https://docs.pymedphys.com/en/latest/contrib/setups/setup-win.html

The GitHub version of PyMedPhys will be scoped under "poetry". So if you run:

```bash
pymedphys gui
```

That will run the released PyPI version that you have installed globally.

If you run the following from within the `pymedphys` GitHub directory:

```bash
poetry run pymedphys gui
```

Then that will explicitly utilise the `pymedphys` within the active GitHub repo.

**Comment by SimonBiggs** [comment]
> When you commit, `pre-commit` should automatically run and fix any issues in the local branch that violate the pre-commit checks

Some additional information. To run pre-commit after a commit has already been made you need to run:

```bash
poetry run pre-commit run --all-files
```

**Comment by SimonBiggs** [comment]
So, I'm going through and doing maintenance of our hanging PRs. For future reference, for other maintainers, here is the process I went through to fix this on behalf of @guanfada 

I installed the GitHub CLI on Windows using winget (first ran winget search GitHub CLI to find the right package):

```
winget install GitHub.cli
```

Then I ran:

```
gh pr checkout 1700
```

Which allowed me to directly check out @guanfada's fork.

I then ran:

```
poetry run pre-commit run --all-files
```

I then ran `git push`, which informed me I needed to run the following:

```
git push git@github.com:guanfada/pymedphys.git HEAD:main
```

**Review by SimonBiggs — APPROVED** [review]

**Pull request merged**